internal/controller/spinapp_controller.go

```diff
@@ -279,7 +307,10 @@ func (r *SpinAppReconciler) reconcileDeployment(ctx context.Context, app *spinv1
 		}
 	}
 
-	desiredDeployment, err := constructDeployment(ctx, app, config, generatedRuntimeConfigSecretName, r.Scheme)
+	// TODO: make this configurable
+	caSecretName := "spin-ca"
```

Comment:
Where do you imagine this being configurable?

Reply:
RuntimeConfig